src/vue/pages/TradeUserOrders.vue

```diff
@@ -0,0 +1,91 @@
+<template>
+  <div class="trade-user-orders">
+    <trade-open-orders
+      :assets="assets"
```

Comment:
here and everywhere: pass quote and base props instead. or rename them to assetPair or something